js/src/lib/interfaces/Types.ts

```diff
@@ -1,3 +1,5 @@
+import type { ModelLibrary } from "./Libraries";
```

Comment:
i would maybe put `LIBRARY_TASK_MAPPING_EXCLUDING_TRANSFORMERS` into a new file to prevent the circular imports between `Libraries.ts` and `Types.ts` (even if it's just types)

Reply:
like for the already existing `TASKS_MODEL_LIBRARIES` which is very similar, BTW (cc @xianbaoqian)

Reply:
(we can change that in the future though – no need to do it in this PR)